editor/animation_track_editor.cpp

```diff
@@ -3424,7 +3424,7 @@ void AnimationTrackEditor::_animation_update() {
 
 	bool same = true;
 
-	if (track_edits.size() == animation->get_track_count()) {
+	if (animation.is_valid() && track_edits.size() == animation->get_track_count()) {
```

Comment:
IMO you can do
```cpp
if (animation.is_null()) {
	return;
}
```
at the start of the method instead.

Reply:
Done!